ksqldb-streams/src/main/java/io/confluent/ksql/execution/streams/StepSchemaResolver.java

```diff
@@ -353,12 +353,14 @@ private LogicalSchema sameSchema(final LogicalSchema schema, final ExecutionStep
     return schema;
   }
 
+  //todo: pipe pseudocolumnVersion here
```

Comment:
What's the comment here for? Is this going to be part of this PR?

Reply:
Just a remnant I didn't catch, removed.